server/IDEs/eclipse/Refine with extensions.launch

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication">
-<listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
```

Comment:
This change is unrelated and should be reverted so that the PR only touches relevant files. Because automatic updates to this file are a recurrent problem I will update it independently.

Reply:
Done.